apps/console/src/public/deployment.config.json

```diff
@@ -1051,10 +1052,19 @@
             "My Account"
         ],
         "legacyFeatures": {
+            "apiResourcesDisabled": false,
+            "applicationListSystemApps": false,
             "applicationOIDCSubjectIdentifierEnabled": false,
             "applicationRequestPathAuthenticationEnabled": false,
             "applicationRoleEnabled": false,
-            "applicationXacmlAuthorizationEnabled": false
+            "applicationSystemAppsSettings": false,
+            "applicationXacmlAuthorizationEnabled": false,
+            "approvalsEnabled": false,
+            "certificatesEnabled": false,
+            "loginAndRegistrationEmailDomainDiscoveryDisabled": false,
+            "organizationsDisabled": false,
```

Comment:
shall we go with `"organizationsEnabled": true` as double negations are confusing to read?

Reply:
The reason we went with `organizationsDisabled` is that the legacyFeatures should be `false` by default and there's a high level config called `legacyMode` which sets all the values to `true`. To maintain this behaviour we need this value even though it's a bit confusing to read.

Reply:
Shall we rename the config from `legacyFeatures` to `legacyMode`? and remove `Enabled` & `Disabled` *suffixes*?

```json
        "legacyMode": {
            "apiResources": true,
            "applicationListSystemApps": false,
            "organizations": true,
            ...
```

Reply:
Ack... renamed the values with f2a36231e2d55e6b9be6108959d3a34872c789e8